src/commands/app/start-services.ts

```diff
@@ -64,22 +64,22 @@ export default class AppStartServices extends Kommand {
     }
 
     this.log(
-      chalk.grey(`\nWriting docker-compose file to ${docoFilename}...\n`)
+      chalk.grey(`\nWriting docker compose file to ${docoFilename}...\n`)
```

Comment:
Just to add proper capitalization. (Docker Compose is the name of a product)

```suggestion
      chalk.grey(`\nWriting the Docker Compose file to ${docoFilename}...\n`)
```